packages/tools/examples/splineROITools/index.ts

```diff
@@ -273,22 +266,7 @@ async function run() {
       },
     ],
   });
-
-  toolGroup.setToolActive(PanTool.toolName, {
-    bindings: [
-      {
-        mouseButton: MouseBindings.Auxiliary, // Middle Click
-      },
-    ],
-  });
-
-  toolGroup.setToolActive(ZoomTool.toolName, {
-    bindings: [
-      {
-        mouseButton: MouseBindings.Secondary, // Right Click
-      },
-    ],
-  });
+  addNavigationBindings(toolGroup);
```

Comment:
I was thinking we call pan, zoom, scroll etc `manipulatoin` in the code base elsewhere and not navigation -> `addManipulationBindings`